resources/features/Sic.feature

```diff
@@ -0,0 +1,226 @@
+Feature: A sic calculation groups the input data by substrings of SIC and ID to determine which subgroup contains the highest amount of employees
+         and assigns a subdivision determined by the division and group.
+
+         The method uses the following substrings:
+         first 2 characters as division,
+         first 3 characters as group,
+         first 4 characters as class,
+         and the whole whole SIC.
+
+    @Happy Path
+    Scenario Outline: Happy Path - Original Happy Path
+        Given input:
+        | ern | lurn | sic07 | employees |
+        | 123 |    1 | 63111 |       504 |
+        | 123 |    4 | 41202 |       844 |
+        | 123 |    6 | 69201 |       188 |
+        | 123 |   10 | 46444 |       301 |
+        | 123 |   13 | 47344 |       300 |
+        | 123 |   14 | 47844 |       900 |
+        | 123 |   15 | 47944 |       950 |
+        | 345 |    2 | 63110 |        46 |
+        | 345 |    7 | 12312 |       100 |
+        | 345 |   11 | 47544 |       500 |
+        | 345 |    9 | 46212 |       300 |
+        | 456 |    8 | 46123 |       500 |
+        | 456 |    3 | 63120 |       540 |
+        | 654 |    5 | 78109 |       517 |
+        | 654 |   12 | 47144 |      1000 |
+        When the Sic method is calculated
```

Comment:
Is the SIC method being triggered for all Enterprises regardless of attached Local Units etc?
If yes, could this be improved by the When clause asserting a check that the Enterprises has >1 Local Units and that the Local Unit SICs are not equal?
This also has the logic of then using the Enterprise frame as an input (for the live LU check), which can then subsequently be written back to as part of the output Then statement.